examples/cpp/demo_config.h

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (2019) Cobalt Speech and Language, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef DEMO_CONFIG_H
+#define DEMO_CONFIG_H
+
+#include "cubic.pb.h"
+
+#include <string>
+
+class DemoConfig
+{
+public:
+    /*
+     * Parse the command-line arguments given to application at startup
+     * and return the configuration based on the args. It is expecting
+     * "-config <path/to/config.toml>" to be included in the arguments and
+     * throws an exception if they are not (or if there are errors parsing).
```

Comment:
Is this true?  I thought I saw somewhere that we look for a default `./config.toml`.

Reply:
Excellent point. I suppose it is half true. If there are no command line args (besides the name of the application itself), it uses a default. If there are arguments, but the config file is not among them, it will throw an exception.